assets/models/system/Deci-DeciCoder-1b/description.md

```diff
@@ -0,0 +1,43 @@
+The Model Card for DeciCoder 1B provides details about a 1 billion parameter decoder-only code completion model developed by Deci. The model was trained on Python, Java, and JavaScript subsets of Starcoder Training Dataset and uses Grouped Query Attention with a context window of 2048 tokens. It was trained using a Fill-in-the-Middle training objective and generated by Deci's proprietary Neural Architecture Search-based technology, AutoNAC. The model is intended for single/multiline code completion from a context window of up to 2048 tokens. The model has limitations as it has undergone training with source code from Python, Java, and JavaScript, and there is no assurance that the resulting code will function as expected. The Model Card provides details on how to use the model, training details, and evaluation results. The model's checkpoints are licensed under the Apache 2.0 license.
```

Comment:
Lets use HF model card details and not use GPT for summarization

Reply:
refer to: [doc](https://microsoftapc-my.sharepoint.com/:w:/g/personal/ayushmishra_microsoft_com/ETjbEy6poUhBsdysFKduH3IBmrdEe2gP2Yzza3p6COAAhA?e=tegVOv) for more details